### DISCUSSION TITLE
Windows Terminal v1.19.10821.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#16933] [Announcements]
This is the March servicing release for Windows Terminal Stable, rolling out *now* via the Store and WinGet!

## Bug Fixes and Changes

* We've rolled back to an older compiler to get Terminal working on Intel Core 2 Duo machines again (!) (#16907)
* `ReadConsoleOutputCharacterW` no longer fails on lines containing narrow surrogate pairs (#16898)
* Auto-detected URLs and search results are no longer pushed around by wide characters (oops) (#16787) (thanks @Comzyh and @lhecker!)
* We will now display control characters literally if they were produced with the Windows Console APIs (#16825) (thanks @j4james!)
* Entering the tab switcher via the command palette no longer selects the wrong tab (#16858) (thanks @AlejandroBlanco2001!)
* All localizations are now checked into the project, so you can build any historic version with all of its original languages (#16835)

---

_Binary files inside the unpackaged distribution archive bear the version number `1.19.240322001`._

### Asset Hashes

- Microsoft.WindowsTerminal_1.19.10821.0_8wekyb3d8bbwe.msixbundle
   - SHA256 `EE0D267D20BF2F48C998DB258E83215F74FE69D9D08A50B9A2DBD008CAE52196`
- Microsoft.WindowsTerminal_1.19.10821.0_8wekyb3d8bbwe.msixbundle_Windows10_PreinstallKit.zip
   - SHA256 `2A433736B0CF6EA215BE6DA0B785520AC950060D50903FBC39EFACB815FD95EC`
- Microsoft.WindowsTerminal_1.19.10821.0_arm64.zip
   - SHA256 `12ED0807BFC21E1DCC41949C070910711A8E891AC783816B384F194ECE645CDF`
- Microsoft.WindowsTerminal_1.19.10821.0_x64.zip
   - SHA256 `B6243B796C57D146D8943828000F86FFE5C9DA0B21A19653C2D9D323706384C1`
- Microsoft.WindowsTerminal_1.19.10821.0_x86.zip
   - SHA256 `6D8BB58109233F819EA1CE833E996151F24A557BDBEBA39A97A51BB7170BE609`

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.19.10821.0'>Windows Terminal v1.19.10821.0</a>.</em>

**Comment by HR320** [comment]
I can`t upgrade with winget to the stable version v1.19.10821.0 (https://github.com/microsoft/terminal/releases/tag/v1.19.10821.0)
Why? I am still using windows 11 21H2.
Thanks!
HR

**Reply by DHowett** [reply]
What error message are you seeing when you try to upgrade?

**Comment by andyc56-mso** [comment]
Is there an active bug report against the 14.38 compiler for this issue? I searched the Microsoft Developer Community c++ bug list and couldn't find one. I'll be releasing some software in May, and was wondering which version of VS2022 I should use for the final build.

**Comment by HR320** [comment]
Hello,
never mind, now I have the right version
Windows Terminal v1.19.10821.0
Thanks

Am Sa., 6. Apr. 2024 um 00:54 Uhr schrieb Andy C ***@***.***>:
